sixtyfps_runtime/interpreter/api.rs

```diff
@@ -1059,6 +1063,7 @@ pub enum SetCallbackError {
 
 /// Error returned by [`ComponentInstance::invoke_callback`]
 #[derive(Debug, Clone, Copy, PartialEq, Eq, thiserror::Error)]
+#[non_exhaustive]
```

Comment:
In a folloup patch, we can add the error if arguments type are incorrect

Reply:
👍 Please do